lib/mixlib/install/cli.rb

```diff
@@ -36,6 +36,10 @@ def list_versions(product_name, channel)
         default: "x86_64",
         aliases: ["-a"],
         enum: Mixlib::Install::Options::SUPPORTED_ARCHITECTURES.map(&:to_s)
+      option :platform_version_compat,
+        desc: "Enable or disable platform version compatibility mode when using platform detection",
```

Comment:
We might want to mention that this is platform version backward compatibility mode. I don't think that is completely obvious as things are worded now.